onnxruntime/core/providers/openvino/backend_manager.cc

```diff
@@ -107,12 +108,14 @@ BackendManager::BackendManager(const GlobalContext& global_context,
                                                       subgraph_context_,
                                                       ep_ctx_handle_);
     } catch (const OnnxRuntimeException& ex) {
+      std::string exception_str = ex.what();
+      bool enable_cpu_fallback = !GetGlobalContext().disable_cpu_fallback;
```

Comment:
When there is a build time flag set, OPENVINO_DISABLE_NPU_FALLBACK it implies GetGlobalContext().disable_cpu_fallback = False. 
But here it will be set to default true.  I think this logic will fail. 

@srirammaswamy-intel  Can you build with OPENVINO_DISABLE_NPU_FALLBACK  directive and validate the exception

Reply:
Afer this assignment, `OPENVINO_DISABLE_NPU_FALLBACK` will overwrite the value of `enable_cpu_fallback`. So `OPENVINO_DISABLE_NPU_FALLBACK` will be taken as high preference.